src/java/main/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -113,13 +113,21 @@ private QuorumServer(long id, InetSocketAddress addr,
             this.id = id;
             this.addr = addr;
             this.electionAddr = electionAddr;
+            String checkIPReachableValue = System.getProperty("zookeeper.checkIPTimeout");
```

Comment:
I think we can improve this naming game just a little bit. Why not call this variable `ipTimeout`? And `checkIPReachableTO` as `ipReachableTimeout` or `reachableIPTimeout`?